lib/membrane_wav/common.ex

```diff
@@ -0,0 +1,10 @@
+defmodule Membrane.WAV.Common do
+  @moduledoc false
+  alias Membrane.Caps.Audio.Raw
+
+  @spec convert_to_demand_in_bytes(integer(), integer(), Raw.t()) :: list
+  def convert_to_demand_in_bytes(buffers_count, frames_per_buffer, caps) do
```

Comment:
```suggestion
  @spec convert_to_demands_in_bytes(integer(), integer(), Raw.t()) :: [Membrane.Element.Action.demand_t()]
  def convert_to_demands_in_bytes(buffers_count, frames_per_buffer, caps) do
```
Of course, you can also alias `Membrane.Element.Action`